app/javascript/article-form/articleForm.jsx

```diff
@@ -264,10 +265,10 @@ export class ArticleForm extends Component {
     });
   };
 
-  toggleModal = () => {
+  toggleModal = (isOpen) => {
```

Comment:
Nitpick (non-blocking): Rename `isOpen` to `showModal` and then it can be used in `setState` like so:

```js
this.setState({
  showModal,
});
```